modules/ROOT/pages/questions/question-11.adoc

```diff
@@ -0,0 +1,13 @@
+11. the following plugins were removed from _TinyMCE 6.0_.
```

Comment:
@EkimChau @metricjs

Reply:
No alternative:
* `fullpage`
* `bbcode`
* `legacyoutput`

Premium offering:
* `spellchecker`



These features are now built in, in fact I think all were built-in on 5 and left as empty plugins?
* `colorpicker`
* `contextmenu`
* `textcolor`

Reply:
> These features are now built in, in fact I think all were built-in on 5 and left as empty plugins?

This is correct, they are all part of core now.